src/js/components/mapWidgets/widgetContent/CanopyDensityContent.tsx

```diff
@@ -52,6 +52,7 @@ const CanopyDensityContent = (): JSX.Element => {
     dispatch(setCanopyDensity(value));
     //send % value to modify the layer
     mapController.updateDensityValue(markValueMap[value]);
+    mapController.updateBiodensityValue(value);
```

Comment:
When I test the UI, it looks like the biolayer isn't updating on the map in comparison to PROD